tests/io/good/subscope.catala_en

```diff
@@ -34,8 +34,8 @@ $ catala Typecheck --check-invariants
 
 ```catala-test-cli
 $ catala Dcalc -s B
-let scope B (B_in: #[test] B_in): #[test] B =
```

Comment:
Why attributes are sometimes removed like here and sometimes added like above?

Reply:
The dcalc is a bit quirky when supplied no scope like in the case above, it'll choose a random scope and print the program after calling `Program.to_expr` on it (hence using the expr let-in printer instead of the scope printer). That's for historical reason and I was afraid to change it at the same time when I refactored the main driver, but I think it can safely be put on par with the lcalc printer now. The removal of the attrs in this case was voluntary.

It may be better to do that change in a follow-up PR to avoid adding noise to the test result changes though ?